app/renderer/src/components/Inspector/Source.jsx

```diff
@@ -20,27 +21,98 @@ const Source = (props) => {
     methodCallInProgress,
     mjpegScreenshotUrl,
     isSourceRefreshOn,
+    pageSourceSearchText,
     t,
   } = props;
 
+  /**
+   * Highlights the part of the node text in source tree that matches the search term.
+   */
+  const highlightNodeMatchingSearchTerm = (nodeText, searchText) => {
+    if (!searchText) {
+      return nodeText;
+    }
+
+    const index = nodeText.toLowerCase().indexOf(searchText.toLowerCase());
```

Comment:
is nodeText always defined?

Reply:
Node Text represents each XML node in the app source. So it's unlikely to be undefined. But i have added a safety check to make sure it is defined before processing.